docs/source/configuration/validation.md

```diff
@@ -291,7 +319,21 @@ export const isNumber = ({ value, formatMessage }: Validator) => {
 Using `formData`, you can perform validation checks using other field data as source.
 This is useful when you want to validate two related fields, such as ensuring the end date of an event is after its start date.
 You can create invariant validator types.
+In the following code snippet you can check how to create a validator method for checking if the Event content type `start`/`end` fields are valid:
```

Comment:
Note that this narrative text reflects that the start and end dates of an event content type are different. I think I read this code sample correctly, but another pair of non-jetlagged eyes would be good.

```suggestion
The following code snippet shows how to create a validator method that ensures the event content type's end date is after its start date.
```